src/_actions/assetTypes.actions.js

```diff
@@ -15,4 +22,31 @@ export const loadAssetTypes = () => (dispatch) => {
     }));
 };
 
-export default loadAssetTypes;
+export const createAssetType = newAssetType =>
+  dispatch => axios.post('asset-types', newAssetType).then((response) => {
+    dispatch(createAssetTypeSuccess(response.data));
+    dispatch(updateToastMessageContent('Asset Type Saved Successfully',
+      'success'));
+  }).catch((error) => {
+    dispatch(createAssetTypeFailure(error));
+    dispatch(updateToastMessageContent('Could Not Save The Asset Type',
+      'error'));
+  });
+
+export const createAssetTypeSuccess = assetType => (
```

Comment:
This is a misuse of `export` keyword. Only export action creators when they are being imported in a different file. The same comment applies to all action creators in this PR

Reply:
Alright makes sense.